### ISSUE TITLE
Clarification of README

### ISSUE THREAD

**Issue by tylerjheslop** [richardk80/spotify-ripper#7]
Do you need someone to write a parser to make 
"https://open.spotify.com/track/3tQ6LmYZeoJ4dwkZ3IGoSj?si=9e7b8b1d2bd1481f" 
above link into below? or below into above?
"spotify:track:3tQ6LmYZeoJ4dwkZ3IGoSj",

**Comment by TimInLasVegas** [comment]
> Do you need someone to write a parser to make "https://open.spotify.com/track/3tQ6LmYZeoJ4dwkZ3IGoSj?si=9e7b8b1d2bd1481f" above link into below? or below into above? "spotify:track:3tQ6LmYZeoJ4dwkZ3IGoSj",

If code for a parser needs to exist here's a basic prototype example of how to split the strings. This code will pass any link through the functions and strip out the string we need and shove it in front of "spotify:".
Input:
python3 spotifyStringSplit.py https://open.spotify.com/track/3zu33333335LR6zwGLB?si=v_PvykSQTxOJXpLP3eIpEQ https://open.spotify.com/album/3zuIMH8P0GKtV5LR6zwGLB?si=v_PvykSQTxOJXpLP3eIpEQ
Output:
spotify:3zu33333335LR6zwGLB
spotify:3zuIMH8P0GKtV5LR6zwGLB
This was written extremely simply so that the concept could be understood by all.

`
import sys
links = sys.argv

def decodeTrack(track):
  track_str = track.split('/track/')[len(track.split('?si')) -1 ].split('?si')[0]
  print ('spotify:' + track_str)
def decodeAlbum(album):
  album_str = album.split('/album/')[len(album.split('?si')) -1 ].split('?si')[0]
  print ('spotify:' + album_str)



for link in range (1, len(sys.argv)):
        lText = sys.argv[link]
        if '/track/' in lText:
         decodeTrack(lText)
        if '/album/' in lText:
         decodeAlbum(lText)
`

**Comment by richardk80** [comment]
@tylerjheslop Yes I do. That would make this easier to use. Getting tired of having to edit the url to a uri. I know other people must be getting tired of doing that too. Will you be able to implement that?
